tempesta_fw/http.c

```diff
@@ -19,6 +19,7 @@
  * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 #include <linux/string.h>
+#include <linux/vmalloc.h>
```

Comment:
You don't use `vmalloc()`, so it's better to use `<linux/gfp.h>` for `__alloc_free_pages()`

Reply:
But `vfree` function is used in `tfw_http_cfg_handle_resp_body` - to release the memory allocated in `tfw_cfg_read_file`.